packages/analytics-browser/src/index.ts

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable @typescript-eslint/unbound-method */
 import client from './browser-client';
-export { createInstance } from './browser-client';
+export { AmplitudeBrowser, createInstance } from './browser-client';
```

Comment:
Why do we need to expose this?

Reply:
I am creating a "analytics-ma-borrowser" package, and it will be extended by MA browser. There is a few issue on ma browser package. I will commit that part later.

Reply:
is is possible to add these changes later when it's actually needed. it's better to keep related changes in the same commit/pr

Reply:
Removed any code related to `analytics-ma-browser` package.